newton/utils/import_mjcf.py

```diff
@@ -54,6 +54,7 @@ def parse_mjcf(
     static_link_mass: float = 1e-2,
     collapse_fixed_joints: bool = False,
     verbose: bool = False,
+    parse_equality: bool = False,
```

Comment:
I think this option should be flipped - by default we parse the constraints if they are there. You could rename it also to `skip_equality_constraints`.